mmv1/third_party/terraform/website/docs/d/compute_region_instance_group_manager.html.markdown

```diff
@@ -0,0 +1,38 @@
+subcategory: "Compute Engine"
+page_title: "Google: google_compute_region_instance_group_manager"
+description: |-
+Get a Compute Region Instance Group within GCE.
+---
+
+# google\_compute\_region\_instance\_group\_manager
+
+Get a Compute Region Instance Group Manager within GCE.
+For more information, see [the official documentation](https://cloud.google.com/compute/docs/instance-groups/distributing-instances-with-regional-instance-groups)
+and [API](https://cloud.google.com/compute/docs/reference/rest/v1/regionInstanceGroupManagers)
+
+## Example Usage
+
+```hcl
+data "google_compute_region_instance_group_manager" "igm1" {
```

Comment:
If there is no more examples then maybe "rigm" would suffice (not "igm1")? or "my-rigm"